packages/manager/modules/notifications-sidebar/src/controller.js

```diff
@@ -77,15 +111,32 @@ export default class NotificationsCtrl {
       });
   }
 
-  getNumberOfActiveNotifications() {
-    return filter(this.sublinks, (notification) => notification.isActive)
-      .length;
+  $postLink() {
+    // Avoid click propagation inside the menu
+    // Since we're binding a click function of the document
+    this.$timeout(() => {
+      this.$element.on('click', (e) => {
+        e.stopPropagation();
+      });
```

Comment:
**nitpick:** Using mangled variable name is not convenient to deal with. Why not prefer using `event` over `e`?

```suggestion
      this.$element.on('click', (event) => {
        event.stopPropagation();
      });
```